pkg/sentry/arch/stack.go

```diff
@@ -185,6 +185,8 @@ func (s *Stack) Load(args []string, env []string, aux Auxv) (StackLayout, error)
 	}
 	l.EnvvStart = s.Bottom
 
+	s.Align(8)
+
```

Comment:
Hi, from the Arm64 AAPCS doc
```
5.2.2.1 Universal stack constraintsAAdditionally, atany point at which memory is accessed via SP, the hardware requires thato
SP mod 16 = 0.  The stack must be quad-word aligned.
```
Shouldn't it be set 16 bytes aligned here?

Reply:
if SCTLR_EL1.SA0 (bit4) is 0, 8 is enough, but for better compatibility, It is better to set it to 16. will update to 16

Reply:
It would be better to add a comment here. And the next code block pushes args on stack and then we align a stack pointer to 32 bytes. Why is this not enough?..

Reply:
Hi avagin:

Yes, you are right.  

This modification is not needed after we have this patch "arm64: the mair_el1 value is wrong"。The original issue I meet in arm64 platform is when libc initialization，it will hang in memcpy function which use an address passed by AUXV, the address is not aligned，and the memory is set as device memory, on armv8 platform "An unaligned access to any type of Device memory causes an Alignment fault". So I apply an original patch like below:

`diff --git a/pkg/sentry/arch/stack.go b/pkg/sentry/arch/stack.go
index 1108fa0..01fd72f 100644
--- a/pkg/sentry/arch/stack.go
+++ b/pkg/sentry/arch/stack.go
@@ -103,6 +103,9 @@ func (s *Stack) Push(vals ...interface{}) (usermem.Addr, error) {
                }
 
                s.Bottom -= usermem.Addr(n)
+               if s.Bottom%usermem.Addr(8) != 0 {
+                       s.Bottom -= (s.Bottom % usermem.Addr(8))
+               }
        }
 
        return s.Bottom, nil
`
After I fix the mair_el1 issue, I update this patch by current one, but actually, this issue is also fixed.